tests/vmcheck/test-override-kernel.sh

```diff
@@ -33,14 +33,14 @@ vm_cmd 'curl -sS -L -O https://dl.fedoraproject.org/pub/fedora/linux/releases/27
 current=$(vm_get_booted_csum)
 vm_cmd rpm-ostree db list "${current}" > current-dblist.txt
 assert_not_file_has_content current-dblist.txt 'kernel-4.13.9-300.fc27'
-grep -E '^ kernel-' current-dblist.txt  | sed -e 's,^ *,,' > current-kernel.txt
+grep -E '^ kernel-4' current-dblist.txt  | sed -e 's,^ *,,' > orig-kernel.txt
```

Comment:
```
+ grep -E '^ kernel-4' current-dblist.txt
+ sed -e 's,^ *,,'
+ _cleanup_tmpdir
+ test -z ''
+ test -f /var/tmp/checkout/tests/vmcheck/test.PCCWYD/.test
+ rm /var/tmp/checkout/tests/vmcheck/test.PCCWYD -rf
FAIL
```

Ahh, right, we should probably just skip this test on CentOS?

Reply:
Funny that we'd been installing the fedora kernel on centos...  Fixup :arrow_dow